docs/getting-started/linting/FAQ.md

```diff
@@ -1,23 +1,11 @@
-# Troubleshooting / FAQ
-
-## Table of Contents
-
-- [I am using a rule from ESLint core, and it doesn't work correctly with TypeScript code](#i-am-using-a-rule-from-eslint-core-and-it-doesnt-work-correctly-with-typescript-code)
-- [I get errors telling me "The file must be included in at least one of the projects provided"](#i-get-errors-telling-me-the-file-must-be-included-in-at-least-one-of-the-projects-provided)
-- [I use a framework (like Vue) that requires custom file extensions, and I get errors like "You should add `parserOptions.extraFileExtensions` to your config"](#i-use-a-framework-like-vue-that-requires-custom-file-extensions-and-i-get-errors-like-you-should-add-parseroptionsextrafileextensions-to-your-config)
-- [One of my lint rules isn't working correctly on a pure JavaScript file](#one-of-my-lint-rules-isnt-working-correctly-on-a-pure-javascript-file)
-- [TypeScript should be installed locally](#typescript-should-be-installed-locally)
-- [How can I ban `<specific language feature>`?](#how-can-i-ban-specific-language-feature)
-- [Why don't I see TypeScript errors in my ESLint output?](#why-dont-i-see-typescript-errors-in-my-eslint-output)
-- [I get errors from the `no-undef` rule about global variables not being defined, even though there are no TypeScript errors](#i-get-errors-from-the-no-undef-rule-about-global-variables-not-being-defined-even-though-there-are-no-typescript-errors)
-- [How do I check to see what versions are installed?](#how-do-i-check-to-see-what-versions-are-installed)
-- [My linting feels really slow](#my-linting-feels-really-slow)
-
+---
+id: troubleshooting
+title: Troubleshooting
+sidebar_label: Troubleshooting
+hide_title: true
 ---
 
-<br />
-<br />
-<br />
```

Comment:
ditto - removing these breaks makes it much harder to read this in the github renderer

Reply:
do we want to focus on documentation on github or on website?